core/src/main/java/org/jdbi/v3/core/BaseStatement.java

```diff
@@ -55,24 +55,36 @@ protected StatementBuilder getStatementBuilder()
         return statementBuilder;
     }
 
-    void addCleanable(Cleanable cleanable)
+    /**
+     * Registers the given {@link Cleanable} to be executed when this statement is closed.
+     *
+     * @param cleanable the cleanable to register
+     * @return this
+     */
+    @SuppressWarnings("unchecked")
+    public final This addCleanable(Cleanable cleanable)
```

Comment:
I think this is a vestigial tail left over from some intermediate work. I'll make it private again.